chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/surveillance/SurveillanceDAO.java

```diff
@@ -935,9 +865,7 @@ private void populateSurveillanceNonconformityEntity(SurveillanceNonconformityEn
         to.setFindings(from.getFindings());
         to.setResolution(from.getResolution());
         to.setSitesPassed(from.getSitesPassed());
-        if (from.getStatus() != null) {
-            to.setNonconformityStatusId(from.getStatus().getId());
-        }
+        to.setNonConformityCloseDate(from.getNonconformityCloseDate());
```

Comment:
I'm having trouble figuring out what case / capitalization we're using on "NonConformity". This line uses both, and I don't know which is right, but we should pick one and stick with it, right?